src/windows/symbol.rs

```diff
@@ -473,3 +474,26 @@ impl RvaSymbols {
         Self::fill_the_gaps(all_syms, ranges)
     }
 }
+
+pub(super) fn symbolic_to_pdb_symbols(syms: PeSymbolIterator) -> PDBSymbols {
+  let mut pdb_syms = PDBSymbols::default();
+
+  for sym in syms {
+    if let Some(name) = sym.name() {
+      let pdb_symbol = PDBSymbol {
+        name: name.to_string(),
```

Comment:
Maybe you should try to use TypeDumper::dump_function  with an Index(0) to have a change to demangle the name and to get the parameter_size.
You can see: https://github.com/mozilla/dump_syms/blob/master/src/windows/symbol.rs#L223.

Reply:
Does that apply to Symbolic symbols too? I assumed that the names would be automatically de-mangled in that case but maybe I assumed wrong.

Reply:
I don't know, need to check, and if parameter_size matters, then there are some heuristics to guess it.
Maybe you can try with dll we've in `test_data` dir and if they aren't insteresting maybe you can try to compile basic.cpp with the good options to have some symbols in the dll itself (see the Makefile in this dir).